### PR TITLE
Add path in find-tar for locating node binary tarball

### PR DESCRIPTION
**Pull request by yujuhong**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
/kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

### CODE REVIEW

File: cluster/common.sh [hunk 1]
```diff
@@ -299,6 +299,7 @@ function set_binary_version() {
 function find-tar() {
   local -r tarball=$1
   locations=(
+    "${KUBE_ROOT}/node/${tarball}"
```
Comment (ixdy) [inline cluster/common.sh]: cause I couldn't remember: this path is coming from https://github.com/kubernetes/kubernetes/blob/5c780c8cb134bad1439e170c4f834586a0bc5373/cluster/get-kube-binaries.sh#L230

**Review by ixdy — COMMENTED** [review]
/lgtm

**Comment by yujuhong** [comment]
@ixdy thanks for the review!

**Pull request merged**